test/unit/HoldConfirmation.test.js

```diff
@@ -378,15 +371,8 @@ describe('HoldConfirmation', () => {
       const main = component.find('main');
 
       expect(main.find('#delivery-location')).to.have.length(1);
-      expect(main.contains(
-        <p id="delivery-location">
-          The item will be delivered to: <span>
-            please <a href="https://gethelp.nypl.org/customer/portal/emails/new">email us</a> or
-            call 917-ASK-NYPL (<a href="tel:19172756975">917-275-6975</a>) for your delivery
-            location.
-          </span>
-        </p>
-      )).to.equal(true);
+      expect(main.find('#delivery-location').text()).to.equal('The item will be delivered to: please email us or call 917-ASK-NYPL (917-275-6975) for your delivery location.');
```

Comment:
So, it's arguably separate from the work of fixing the EDD language, but am I reading this correctly there's a unit test asserting that the confirmation say "The item will be delivered to:" without subsequently rendering the delivery location? That's definitely the behavior I've seen using it, but it seems like a bug, right? Again, separate from this ticket, but surprising there would be an existing unit test asserting broken language.

Reply:
Yes. There is a test for that. It's if the pickup location is missing or does not match any of the actual locations. This might have been the chosen behavior awhile ago. There are plenty of oddities in SCC.